app/models/course.rb

```diff
@@ -127,6 +127,10 @@ def new_draft_attributes
     end.order(:changed_at, :id)
   end
 
+  scope :not_new, -> do
+    includes(site_statuses: %i[site course]).where.not(SiteStatus.table_name => { status: 'N' })
```

Comment:
Not sure about this syntax. Doesn't look very "railsy" 🤔

Reply:
The `N` should be using the site_status enum:
https://github.com/DFE-Digital/manage-courses-backend/blame/edit-options/app/models/site_status.rb#L35

Reply:
Resolved in #645